src/components/Contracts/index.tsx

```diff
@@ -0,0 +1,236 @@
+      <Translate id="playground.codesandbox.description">
+        Module to make interacting with a smart contract easier, without having
+        to resolve through its owner first.
+      </Translate>
+    ),
+  },
+  {
+    name: '⛽ LSP25 Execute Relay Call',
+    url: '',
+    urlABI: 'https://docs.lukso.tech/contracts/contracts/LSP25ExecuteRelayCall',
+    description: (
+      <Translate id="playground.codesandbox.description">
+        Add Meta Transactions on your contract to enable gas-less transactions
+        and more easily onboard new users.
+      </Translate>
+    ),
+  },
+];
+
+const LSPs_DigitalAssets = [
+  {
+    name: '🔍 LSP4 Digital Asset Metadata',
+    url: 'https://docs.lukso.tech/contracts/overview/DigitalAssets/#lsp4-digital-asset-metadata',
+    urlABI: 'https://docs.lukso.tech/contracts/contracts/LSP6KeyManager',
+    description: (
+      <Translate id="playground.codesandbox.description">
+        Enables to add metadata to any digital asset contract (token or NFT).
+      </Translate>
+    ),
+  },
+  {
+    name: '🪙 LSP7 Digital Asset',
+    url: 'https://docs.lukso.tech/contracts/overview/DigitalAssets/#lsp7-digital-asset',
+    urlABI:
+      'https://docs.lukso.tech/contracts/contracts/LSP7DigitalAsset/LSP7DigitalAsset.md',
+    description: (
+      <Translate id="playground.codesandbox.description">
+        Creates your own fungible token with the possibility to make it
+        divisible or not.
+      </Translate>
+    ),
+  },
+  {
+    name: '🎨 LSP8 Identifiable Digital Asset',
+    url: 'https://docs.lukso.tech/contracts/overview/DigitalAssets/#lsp8-identifiable-digital-asset',
+    urlABI:
+      'https://docs.lukso.tech/contracts/contracts/LSP8IdentifiableDigitalAsset/LSP8IdentifiableDigitalAsset.md',
+    description: (
+      <Translate id="playground.codesandbox.description">
+        Creates a non-fungible token where each NFT can be represented
+        differently (numbers, serial numbers, an ERC725Y contract per NFT...).
+      </Translate>
+    ),
+  },
+];
+
+const LSPs_OwnershipAccessControl = [
+  {
+    name: '🔐 LSP6 Key Manager',
+    url: 'https://docs.lukso.tech/contracts/overview/KeyManager',
+    urlABI: 'https://docs.lukso.tech/contracts/contracts/LSP6KeyManager',
+    description: (
+      <Translate id="playground.codesandbox.description">
+        Allows multi-control over the account using different permissions.
+      </Translate>
+    ),
+  },
+  {
+    name: '🌱 LSP11 Basic Social Recovery',
+    url: '',
+    urlABI:
+      'https://docs.lukso.tech/contracts/contracts/LSP11BasicSocialRecovery/',
+    description: (
+      <Translate>
+        Recover access to your Blockchain based account if you are locked using
+        a social recovery mechanism.
+      </Translate>
+    ),
+  },
+  {
+    name: '2️⃣ LSP14 Ownable 2 Steps',
+    url: '',
+    urlABI: 'https://docs.lukso.tech/contracts/contracts/LSP14Ownable2Step/',
+    description: (
+      <Translate>
```

Comment:
I would not put JSX in this object, only the plain description. And if needed, add this "Translate" component in the contractCard rather than here